src/solvers/solveCobraLPCPLEX.m

```diff
@@ -117,7 +117,10 @@
 if ~exist('minNorm','var')
     minNorm=0;
 end
-
+if ~printLevel && isunix

```

Comment:
this addition has been tested and working.
Regarding the test on the OS, I redirect you to the commit description.